Mobile/ShopLens/ShopLensWeb/ShopLensContext.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using Microsoft.EntityFrameworkCore;
+
+namespace ShopLensWeb
+{
+    public class ShopLensContext : DbContext
+    {
+        private bool UseLocalDb { get; }
+        private string ConnectionString { get; set; }
+        
+        public ShopLensContext(string connectionString, bool useLocalDb = false)
+        {
+            UseLocalDb = useLocalDb;
+            ConnectionString = connectionString;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (UseLocalDb)
```

Comment:
Okay, so if "UseLocalDb" is set to TRUE, then a local database is used. You should change the misleading README text then.